servers/audio_server.h

```diff
@@ -155,7 +160,10 @@ class AudioServer : public Object {
 	};
 
 	enum {
-		AUDIO_DATA_INVALID_ID = -1
+		AUDIO_DATA_INVALID_ID = -1,
+		MAX_CHANNELS_PER_BUS = 4,
+		MAX_BUSES_PER_PLAYBACK = 6,
+		LOOKAHEAD_BUFFER_SIZE = 32
```

Comment:
```suggestion
		LOOKAHEAD_BUFFER_SIZE = 32,
```
Trailing comma can ensure that future additions won't generate unnecessary line diff.